backend/src/modules/boards/controller/boards.controller.ts

```diff
@@ -312,4 +316,38 @@ export default class BoardsController {
 
 		return result;
 	}
+
+	@ApiOperation({ summary: 'Update a specific column from a board' })
+	@ApiParam({ type: String, name: 'boardId', required: true })
+	@ApiBody({ type: ColumnDto })
+	@ApiOkResponse({
+		type: BoardDto,
+		description: 'Column updated successfully!'
+	})
+	@ApiBadRequestResponse({
+		description: 'Bad Request',
+		type: BadRequestResponse
+	})
+	@ApiUnauthorizedResponse({
+		description: 'Unauthorized',
+		type: UnauthorizedResponse
+	})
+	@ApiNotFoundResponse({
+		type: NotFoundResponse,
+		description: 'Not found!'
+	})
+	@ApiForbiddenResponse({
+		description: 'Forbidden',
+		type: ForbiddenResponse
+	})
+	@ApiInternalServerErrorResponse({
+		description: 'Internal Server Error',
+		type: InternalServerErrorResponse
+	})
+	@BoardUser([BoardRoles.RESPONSIBLE, TeamRoles.ADMIN, TeamRoles.STAKEHOLDER])
+	@UseGuards(BoardUserGuard)
+	@Put(':boardId/column')
```

Comment:
```suggestion
	@Put(':boardId/column/:columnId')
```
I know you are sending the columnId in the dto but I think the columnId should be included in the route as I'm not mistaken it is a best practice